### ISSUE TITLE
Exported Channel Set fails on Import

### ISSUE THREAD

**Issue by CreativePoweredGeek** [ExpressionEngine/ExpressionEngine#2395]
**Description of the problem**
I exported a channel set out of my local environment for  EE7.1.5 site and when I import it into my staging site I get the error below

**How To Reproduce**

1. export a Channel Set from one EE 7.1.5 site
2.Import into another EE 7.1.5 site
3. See error below

**Error Messages**

<pre><code>
Error Caught
Call to undefined function ExpressionEngine\Library\Filesystem\Adapter\exec()
ee/ExpressionEngine/Library/Filesystem/Adapter/Local.php:161

Stack Trace: Please include when reporting this error
#0 ee/ExpressionEngine/Library/Filesystem/Adapter/Local.php(136): ExpressionEngine\Library\Filesystem\Adapter\Local->attemptFastDelete('...')
#1 ee/vendor-build/league/flysystem-cached-adapter/src/CachedAdapter.php(135): ExpressionEngine\Library\Filesystem\Adapter\Local->deleteDir('home/rkernkt/pu...')
#2 ee/vendor-build/league/flysystem/src/Filesystem.php(208): ExpressionEngine\Dependency\League\Flysystem\Cached\CachedAdapter->deleteDir('home/rkernkt/pu...')
#3 ee/ExpressionEngine/Library/Filesystem/Filesystem.php(260): ExpressionEngine\Dependency\League\Flysystem\Filesystem->deleteDir('home/rkernkt/pu...')
#4 ee/ExpressionEngine/Library/Filesystem/Filesystem.php(224): ExpressionEngine\Library\Filesystem\Filesystem->deleteDir('home/rkernkt/pu...')
#5 ee/ExpressionEngine/Service/ChannelSet/Set.php(177): ExpressionEngine\Library\Filesystem\Filesystem->delete('home/rkernkt/pu...')
#6 ee/ExpressionEngine/Controller/Channels/Sets.php(195): ExpressionEngine\Service\ChannelSet\Set->cleanUpSourceFiles()
#7 [internal function]: ExpressionEngine\Controller\Channels\Sets->doImport()
#8 ee/ExpressionEngine/Core/Core.php(268): call_user_func_array(Array, Array)
#9 ee/ExpressionEngine/Core/Core.php(124): ExpressionEngine\Core\Core->runController(Array)
#10 ee/ExpressionEngine/Boot/boot.php(184): ExpressionEngine\Core\Core->run(Object(ExpressionEngine\Core\Request))
#11 admin.php(153): require_once('...')
#11 admin.php(153): require_once('...')
</code></pre>

**Screenshots / Videos / Template Code**

**Environment Details:**
 - Version: 7.1.5
 - PHP Version 8.1.1 local, 8.0.0 Online
 - MySQL Version [e.g. 5.7]
 - OS: MAc OS browsers, Apache Webserver
 - Web Server:  Apache

**Possible Solution**

**Additional context**
Local & Online are different PHP8 versions

**Comment by CreativePoweredGeek** [comment]
I tried a different Channel set... and while the error wasn't the same I did still get an error:
```Warning
Undefined array key "rte"
ee/ExpressionEngine/Addons/rte/ft.rte.php, line 108

Severity: E_WARNING
Warning
Undefined array key "rte"
ee/ExpressionEngine/Addons/rte/ft.rte.php, line 108

Severity: E_WARNING
Warning
Undefined array key "rte"
ee/ExpressionEngine/Addons/rte/ft.rte.php, line 108

Severity: E_WARNING
Warning
Cannot modify header information - headers already sent by (output started at ee/legacy/core/Exceptions.php:120)
ee/ExpressionEngine/Boot/boot.common.php, line 476

Severity: E_WARNING
Error Caught
Call to undefined function ExpressionEngine\Library\Filesystem\Adapter\exec()
ee/ExpressionEngine/Library/Filesystem/Adapter/Local.php:161

Stack Trace: Please include when reporting this error
#0 ee/ExpressionEngine/Library/Filesystem/Adapter/Local.php(136): ExpressionEngine\Library\Filesystem\Adapter\Local->attemptFastDelete('...')
#1 ee/vendor-build/league/flysystem-cached-adapter/src/CachedAdapter.php(135): ExpressionEngine\Library\Filesystem\Adapter\Local->deleteDir('home/rkernkt/pu...')
#2 ee/vendor-build/league/flysystem/src/Filesystem.php(208): ExpressionEngine\Dependency\League\Flysystem\Cached\CachedAdapter->deleteDir('home/rkernkt/pu...')
#3 ee/ExpressionEngine/Library/Filesystem/Filesystem.php(260): ExpressionEngine\Dependency\League\Flysystem\Filesystem->deleteDir('home/rkernkt/pu...')
#4 ee/ExpressionEngine/Library/Filesystem/Filesystem.php(224): ExpressionEngine\Library\Filesystem\Filesystem->deleteDir('home/rkernkt/pu...')
#5 ee/ExpressionEngine/Service/ChannelSet/Set.php(177): ExpressionEngine\Library\Filesystem\Filesystem->delete('home/rkernkt/pu...')
#6 ee/ExpressionEngine/Controller/Channels/Sets.php(162): ExpressionEngine\Service\ChannelSet\Set->cleanUpSourceFiles()
#7 [internal function]: ExpressionEngine\Controller\Channels\Sets->doImport()
#8 ee/ExpressionEngine/Core/Core.php(268): call_user_func_array(Array, Array)
#9 ee/ExpressionEngine/Core/Core.php(124): ExpressionEngine\Core\Core->runController(Array)
#10 ee/ExpressionEngine/Boot/boot.php(184): ExpressionEngine\Core\Core->run(Object(ExpressionEngine\Core\Request))
#11 admin.php(153): require_once('...')
#11 admin.php(153): require_once('...')```

**Comment by CreativePoweredGeek** [comment]
Sorry to spam my own thread...  I also just had my PHP version updated to the latest (8.1.4 to match my local env)  and the issue persists

**Comment by intoeetive** [comment]
@CreativePoweredGeek thank you for the report.

It looks like `exec` is not available on your system. You should be able to enable it in `php.ini`, or talk to your hosting provider about enabling it.

Not sure about 'rte' error though. Just to let you know, we're working on another system which will replace Channel Sets - but it's also important to know where it fails, so we don't step into same pit holes.

**Comment by CreativePoweredGeek** [comment]
@intoeetive Everything I have read says that enabling exec is a security threat...  is it safe to do this?  I have never needed to have this enabled before.

**Comment by CreativePoweredGeek** [comment]
Also, I checked with my hosting... they say it is...

**Comment by intoeetive** [comment]
@CreativePoweredGeek give a try to to just-released 7.1.6 version. It should solve the `exec` error
